docs/indyav/onboarding.rst

```diff
@@ -0,0 +1,26 @@
+Getting Started on Indy AV
+==========================
+
+Collaboration
+-------------
+
+Slack
+*****
+Join the slack to keep updated on the developement at `uf-cimar <uf-cimar.slack.com>`_.
+Any valid @ufl.edu email will be able to create an account.
+
+Github
+******
+If you see this documentation you are probably already aware of the repository
```

Comment:
``If you see this documentation you are probably already aware of the repository`` -> ``All work (except CAD) and documentation for Indy AV will go in the `MIL REPO <https://github.com/uf-mil/mil>`_.``